fastparquet/writer.py

```diff
@@ -223,7 +224,7 @@ def infer_object_encoding(data):
         return 'json'
     elif all(isinstance(i, bool) for i in head):
         return 'bool'
-    elif all(isinstance(i, int) for i in head):
+    elif all(isinstance(i, (int, long)) for i in head):
```

Comment:
Python 2.7 has `long`s, and this test now passes,  but I probably broke it for Python 3. 

I suspect `fastparquet` is meant for Python3.  I was a fool and installed Python 3.6,I should have known that Numpy is crashy.